### PR TITLE
Corrections Intro.ipynb and Intro_es.ipynb

### PR DESCRIPTION
**Pull request by ChavezUriel**
@alxogm Can you take a look to these corrections?

### CODE REVIEW

File: README.md
Comment (alxogm) [inline README.md:36]: @michaelJwilson I'm not sure if the links are broken on purpose? or if before they were working well as they were... Do you agree with this being fixed?

**Review by alxogm — APPROVED** [review]
Apart from my question to Mike, I think this can be merged. 

(I'm not sure why I can't tag Uriel, but thanks to him for this corrections)

File: README.md
Comment (michaelJwilson) [reply README.md:36]: yeh, this is fine.  Not sure I'll wild about pointing there, but it shouldn't be an issue.

**Comment by michaelJwilson** [comment]
@ChavezUriel Can you perhaps change https://github.com/ChavezUriel/desihigh/blob/master/Espanol/CODE_OF_CONDUCT_es.md

to point to the original https://github.com/michaelJwilson/...  Less likely to run into problems. 
Thanks for the fixes!

**Review by alxogm — APPROVED** [review]

**Comment by michaelJwilson** [comment]
@ChavezUriel @alxogm Apologies for the annoying timing, but I've recently updated the English README and you _might_ want to make similar changes in this PR (entirely) your call.

**Comment by alxogm** [comment]
> @ChavezUriel @alxogm Apologies for the annoying timing, but I've recently updated the English README and you _might_ want to make similar changes in this PR (entirely) your call.

I have gone through the changes to incorporate them (thanks for the explicit mention :D), and resolved a conflict that was present in the English version. I think is ready to merge if you approve @michaelJwilson .

**Comment by michaelJwilson** [comment]
Couldn't track the conflict, but I trust you :) thanks!

**Pull request merged**
